pkgs/top-level/all-packages.nix

```diff
@@ -18413,6 +18413,8 @@ with pkgs;
     vte = gnome3.vte-ng;
   };
 
+  termtosvg = callPackage ../tools/misc/termtosvg { };
```

Comment:
Use `python3Packages.callPackage` and list all packages as input to the function instead.
You can look at `aws_shell` for an example.

Reply:
I thought that `callPackage` is fine in this case if I understand threads like this correctly: https://github.com/NixOS/nixpkgs/pull/38028#issuecomment-377682877

Reply:
Yeah you are right. Sorry for the noise.